grpc-client-autoconfigurer/src/main/java/org/springframework/boot/autoconfigure/grpc/client/DiscoveryClientNameResolver.java

```diff
@@ -52,17 +53,17 @@ public void start(Listener listener) {
 		refresh();
 	}
 
-	@Override
-	public void refresh() {
-		List<ResolvedServerInfo> servers = new ArrayList<ResolvedServerInfo>();
-		System.out.println(client.description());
-		for (ServiceInstance serviceInstance : client.getInstances(name)) {
-			servers.add(new ResolvedServerInfo(InetSocketAddress.createUnresolved(serviceInstance.getHost(), serviceInstance.getPort()),Attributes.EMPTY));
-		}
-		List<List<ResolvedServerInfo>> serversList = new ArrayList<List<ResolvedServerInfo>>(1);
-		serversList.add(servers);
-		this.listener.onUpdate(serversList, Attributes.EMPTY);
-	}
+  @Override
+  public void refresh() {
+    List<List<ResolvedServerInfo>> serversList = new ArrayList<>();
+    for (ServiceInstance serviceInstance : client.getInstances(name)) {
+      List<ResolvedServerInfo> servers = new ArrayList<>();
+      System.out.println("Service Instance: " + serviceInstance.getHost() + ":" + serviceInstance.getPort());
```

Comment:
Hi~, Ray. most other files, they are indented with 2 blank space.  I suppose you  configured your intellij to auto refactor and indent by 2 blank, and I configure it so.
there are diffrent indent type in this project, which do you prefer???

And should I re-post a new pull request??
BTW.   I found that load balancing is not good enough for production environment, still working on it.